beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -50,6 +53,13 @@ const WDInfoDisplay: React.FC<WDInfoDisplayProps> = function ({
     device === Device.MOBILE ||
     device === Device.MOBILE_LG;
 
+  const { phaseMinutes, phaseMinutesRB } = useAppSelector(gameOverview);
+  let phaseLengthInfo = `${getFormattedTime(phaseMinutes * 60)} / phase`;
+  if (phaseMinutesRB !== -1) {
+    phaseLengthInfo = `${phaseLengthInfo} (M) | ${getFormattedTime(
```

Comment:
Could you roll this into the `getFormattedTime` function and avoid the need for the `let`?

In general, there's calculation here that feels like it should belong inside a formatter function rather than inside the render function. You could extract the whole thing to a `getPhaseLengthInfo`

Another aside - I think it's good practice to avoid using the prefix `get` if it's not actually getting anything out of a state / object (for example, `getFormattedTime` implies that the time has already been formatted, not that this function is doing it). For a formatter, I would personally use just `formatTime` / `formatPhaseLengthInfo` etc.